src/api/users/docs/FAQ.md

```diff
@@ -0,0 +1,20 @@
+# FAQ
+
+This document explains several frequently answered questions related to the Users Micro-Service and Firestore. See [`README.md`](https://github.com/Seneca-CDOT/telescope/blob/master/src/api/users/README.md) for more information.
```

Comment:
I'm not an English expert, but doesn't FAQ stand for frequently _asked_ questions?

Reply:
Yes! I wasn't sure what else to call this document besides "frequently asked questions." It's just an internet slang for "a place where frequently asked questions occur".

Reply:
I see XD

Reply:
@chrispinkney  I think Yuan is referring to the line "This document explains several frequently `answered` questions..."

Reply:
🤦 I was uh... sleep deprived... yeah that's why I didn't notice it the first time.

Reply:
I feel you my friend